src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -25,6 +26,14 @@
 
 (defmulti notification-component :type)
 
+;; TODO(rasom): should be removed as soon as all notifications types covered
+(defmethod notification-component :default
```

Comment:
FYI: Although I wrote the solution with a multimethod, it'll die soon. In the branch I'm working on right now I had to split different notifications into different files for two reasons: code conflicts because we're touching the same file... and second and most important, because the identity verification notification has to be rendered also in a bottom sheet.